test/CreatorToken.t.sol

```diff
@@ -20,8 +20,8 @@ contract CreatorTokenTest is Test {
   string PAY_TOKEN_NAME = "Payment Token";
   string PAY_TOKEN_SYMBOL = "PAY";
   uint256 BASE_PAY_AMOUNT = 1e18; // Because our test token has 18 decimals
-  uint256 constant CREATOR_FEE = 700; // 7%
-  uint256 constant ADMIN_FEE = 300; // 3%
+  uint256 CREATOR_FEE = 700; // 7%
+  uint256 ADMIN_FEE = 300; // 3%
```

Comment:
nit: now that these are no longer constants, we can/should change the naming convention and remove the default values

Reply:
fixed in e28ac5b30af2bb086482debf189123a96e11d220